lib/Cmts.ml

```diff
@@ -138,7 +138,7 @@ end = struct
               ( loc.loc_start.pos_lnum - prev.loc_end.pos_lnum
               , next.loc_start.pos_lnum - loc.loc_end.pos_lnum )
             with
-            | 0, 0 -> impossible "already tested by previous if"
+            | 0, 0 -> `Before_next
```

Comment:
I couldn't track where exactly the exception was caught and how it results in a stabilization error. Do you have an idea ?

Reply:
This was introduced in #1509 I just thought this case wouldn't be possible (that's just an assert), but the code of #1449 raises this exception, we just hadn't tested it since #1509 has been merged.

Reply:
My question was where is this exception caught ? I couldn't find how this results in the exact error in https://github.com/ocaml-ppx/ocamlformat/issues/1449

Reply:
This exception is not caught, we get "  Please report this bug at https://github.com/ocaml-ppx/ocamlformat/issues.
  BUG: unhandled exception. Use [--debug] for details." when trying to format the code of #1449 with master (the error message in te issue is not up-to-date)

Reply:
Thanks